src/Uno.UI.RuntimeTests/MUX/Microsoft_UI_Xaml_Controls/RatingControl/RatingControlTests.cs

```diff
@@ -0,0 +1,136 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using MUXControlsTestApp.Utilities;
+
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media.Imaging;
+using Common;
+
+#if USING_TAEF
+using WEX.TestExecution;
+using WEX.TestExecution.Markup;
+using WEX.Logging.Interop;
+#else
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.VisualStudio.TestTools.UnitTesting.Logging;
+#endif
+
+using RatingControl = Microsoft.UI.Xaml.Controls.RatingControl;
+using RatingItemFontInfo = Microsoft.UI.Xaml.Controls.RatingItemFontInfo;
+using RatingItemImageInfo = Microsoft.UI.Xaml.Controls.RatingItemImageInfo;
+using Private.Infrastructure;
+
+namespace Windows.UI.Xaml.Tests.MUXControls.ApiTests
+{
+	[TestClass]
+	public class RatingControlTests : MUXApiTestBase
+	{
+		[TestMethod]
+		public async Task VerifyDefaultsAndBasicSetting()
+		{
+			RatingControl ratingControl = null;
+			RunOnUIThread.Execute(() =>
+			{
+				ratingControl = new RatingControl();
+				Verify.IsNotNull(ratingControl);
+
+				Verify.AreEqual(ratingControl.Caption, "");
+				Verify.AreEqual(ratingControl.InitialSetValue, 1);
+				Verify.AreEqual(ratingControl.IsClearEnabled, true);
+				Verify.AreEqual(ratingControl.IsReadOnly, false);
+				Verify.AreEqual(ratingControl.MaxRating, 5);
+				Verify.AreEqual(ratingControl.PlaceholderValue, -1);
+				Verify.AreEqual(ratingControl.Value, -1);
+
+				// Disabled due to 12359255 reliability issue
+				// Verify.IsTrue(ratingControl.ItemInfo is RatingItemFontInfo, "Verify default type of ItemInfo");
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6162540)